assets/app/view/spreadsheet.rb

```diff
@@ -112,9 +112,13 @@ def render_corporation(corporation)
 
       if !corporation.floated?
         props[:style]['background-color'] = 'rgba(220,220,220,0.4)'
-      elsif (color = StockMarket::COLOR_MAP[corporation.share_price.color])
-        m = color.match(/#(..)(..)(..)/)
-        market_props[:style]['background-color'] = "rgba(#{m[1].hex},#{m[2].hex},#{m[3].hex},0.4)"
+      else
+        share_color = corporation.share_price.color
+        if @game.class::CERT_LIMIT_COLORS.include?(share_color)
+          color = StockMarket::COLOR_MAP[share_color]
+          m = color.match(/#(..)(..)(..)/)
```

Comment:
is there a better way to do this

Reply:
If you mean looking up `share_color` in `CERT_LIMIT_COLORS`, that seems like a pretty reasonable way to find out if a share counts toward the cert limit. I could abstract it into a method in `Game` or something.

If you mean parsing the `StockMarket`'s color and then recalculating it so we can change the alpha, that was there already. I could look into it though.

If you mean a third thing let me know!

Reply:
@jenf has now extracted the match method in lib::Color, so yes i meant the first

Reply:
Better now. Also added a question mark to `counts_for_limit?` since it's a predicate.